ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -161,3 +161,6 @@ sunbird_contextual_help_config={{ contextual_help_config }}
 sunbird_accessibility_guidelines_url={{ sunbird_accessibility_guidelines_url | default("") }}
 sunbird_collection_children_limit={{ sunbird_collection_children_limit | default(1200) }}
 sunbird_questionset_children_limit={{ sunbird_questionset_children_limit | default(500) }}
+
+#release-4.7.0
+allowed_framework_types={{ allowed_framework_types | default("K-12,TPD") }}
```

Comment:
Hi, it'd be helpful if you add a comment, what's the use of this variable, and what are the allowed values?